src/services/SessionEventProcessor.ts

```diff
@@ -43,7 +43,8 @@ export class SessionEventProcessor {
 	}
 
 	async processRequest(sessionEvent: any): Promise<void> {
-		const sessionEventData: SessionEvent = SessionEvent.parseRequest(JSON.stringify(sessionEvent));
+		let sessionEventData;
+		sessionEventData = ExtSessionEvent.parseRequest(JSON.stringify(sessionEvent));
```

Comment:
[nit] this can be done in one line 

`let sessionEventData = ExtSessionEvent.parseRequest(JSON.stringify(sessionEvent));`